src/pages/components/checkbox.mdx

```diff
@@ -1,15 +1,183 @@
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+A Checkbox’s label is part of its touch target.
+
+import Default from '../../examples/forms/checkbox/Default.tsx';
+import DefaultCode from '!!raw-loader!../../examples/forms/checkbox/Default.tsx';
+
+<CodeExample code={DefaultCode}>
+  <Default />
+</CodeExample>
+
+### Regular weight label
+
+Checkbox labels are emphasized by default but can be regular weight.
+
+import RegularWeightLabel from '../../examples/forms/checkbox/RegularWeightLabel.tsx';
+import RegularWeightLabelCode from '!!raw-loader!../../examples/forms/checkbox/RegularWeightLabel.tsx';
+
+<CodeExample code={RegularWeightLabelCode}>
+  <RegularWeightLabel />
+</CodeExample>
+
+### Indeterminate
+
+If a checkbox has child checkboxes, it should show an indeterminate state (reflected by a dash) when
+the child checkboxes contains both some checked and unchecked checkboxes.
+
+import Indeterminate from '../../examples/forms/checkbox/Indeterminate.tsx';
+import IndeterminateCode from '!!raw-loader!../../examples/forms/checkbox/Indeterminate.tsx';
+
+<CodeExample code={IndeterminateCode}>
+  <Indeterminate />
+</CodeExample>
+
+### Hint text
+
+Hint adds additional context to the Checkbox label.
+
+import Hint from '../../examples/forms/checkbox/Hint.tsx';
+import HintCode from '!!raw-loader!../../examples/forms/checkbox/Hint.tsx';
+
+<CodeExample code={HintCode}>
+  <Hint />
+</CodeExample>
+
+### Validation
+
+Success, warning, or danger validation messages are shown using the Message component with the
+Checkbox.
+
+import Validation from '../../examples/forms/checkbox/Validation.tsx';
+import ValidationCode from '!!raw-loader!../../examples/forms/checkbox/Validation.tsx';
+
+<CodeExample code={ValidationCode}>
+  <Validation />
+</CodeExample>
+
+### Hidden
+
+Checkbox labels can be hidden.
+
+import Hidden from '../../examples/forms/checkbox/Hidden.tsx';
+import HiddenCode from '!!raw-loader!../../examples/forms/checkbox/Hidden.tsx';
+
+<CodeExample code={HiddenCode}>
+  <Hidden />
+</CodeExample>
+
+### Disabled
+
+Disabled Checkboxes are not interactive, and not perceived by a screen reader.
+
+import Disabled from '../../examples/forms/checkbox/Disabled.tsx';
+import DisabledCode from '!!raw-loader!../../examples/forms/checkbox/Disabled.tsx';
+
+<CodeExample code={DisabledCode}>
+  <Disabled />
+</CodeExample>
+
+## How to use it well
+
+### Use to select any number of options
+
+<BestPractice>
+  <Do imageSource={props.data.checkboxSelectionDo.children[0].publicURL}>
+    <p>Use to select multiple options from a set of two or more.</p>
+    <p>
+      Use to turn a standalone setting on and off, as a favorable alternative to a{' '}
+      <a href="./toggle">Toggle</a>.
+    </p>
+  </Do>
+  <Dont imageSource={props.data.checkboxStandaloneOptionDont.children[0].publicURL}>
+    <p>
+      Use to select exactly one option from a set of two or more. Use <a href="./radio">Radios</a>
+      instead.
```

Comment:
Missing space. Also, similar comment re: "Radios" vs. "Radio" (defer to word folks).

Reply:
Fixed this invalid href and missing space. 

The text was copied from the reviewed Google Doc. Should it change?

cc: @vsri1137 @rossmoody

Reply:
I was thinking of the current garden page which calls them `Radios`. Let's change to:

> To give the user a mutually exclusive choice use the [Radio] component instead

Reply:
Updated 👍